src/style.css

```diff
@@ -1 +1,89 @@
-/* Styles go here */
+.moyoBox {
+  display: flex;
+  max-width: 1200px;
+  min-width: 1024px;
+  height: 60px;
+  flex-direction: row-reverse;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.text {
+  height: 14px;
+}
+
+.box {
+  padding-top: 15px;
+  padding-bottom: 15px;
+  margin-top: auto;
+  margin-bottom: auto;
+  margin-right: 20px;
+  display: inline-block;
+  font-family: "Cinzel", sans-serif;
+  font-style: medium;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.lastBox {
+  margin-right: 50px;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.imgBox {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  display: flex;
+  flex-grow: 1;
+  margin-left: 50px;
+}
+
+.box:hover {
+  cursor: pointer;
+  color: #00acdc;
+}
+
+[data-qa] {
+  cursor: pointer;
+  color: #00acdc;
+}
+
+.active::after {
+  cursor: default;
+  content: "";
+  position: relative;
+  top: 18.5px;
+  display: block;
+  border-bottom: #00acdc solid 4px;
+  border-radius: 8px;
+  opacity: 0;
+}
+
+.active:active::after {
+  opacity: 1;
+}
```

Comment:
you shouldn't style only link with active class. Every link should have styles on hover but not on click in this task. Class active is only for tests